coops/titans_stronghold.tex

```diff
@@ -186,48 +181,31 @@ \subsection*{\MakeUppercase{Additional Rules}}
     3\bronze 8\silver 10\golden 6\azure}\\
   \end{tabularx}
 }
-
 \vspace*{\fill}
 
 \begin{center}
   \transparent{0.3}{\includegraphics[width=0.6\linewidth, keepaspectratio]{\art/land_mine.png}}
 \end{center}
-
 \vspace*{\fill}
 
 \newpage
 
-\begin{minipage}{0.4\paperwidth}
+\begin{tikzpicture}[overlay, every node/.style={anchor=base}]
   \centering
-  \includegraphics[width=0.18\paperwidth]{\maps/titans-1.png}
-  \captionof{figure}{\textbf{1-PLAYER SCENARIO}}
-\end{minipage}
-\begin{minipage}{0.4\paperwidth}
-  \centering
-  \includegraphics[width=0.36\paperwidth]{\maps/titans-2.png}
-  \captionof{figure}{\textbf{2-PLAYER SCENARIO}}
-\end{minipage}
-\vspace{1em}
-\linebreak
-\begin{minipage}{0.4\paperwidth}
-  \centering
-  \includegraphics[width=0.38\paperwidth]{\maps/titans-3.png}
-  \captionof{figure}{\textbf{3-PLAYER SCENARIO}}
-\end{minipage}
-\begin{minipage}{0.4\paperwidth}
-  \centering
-  \includegraphics[width=0.38\paperwidth]{\maps/titans-4.png}
-  \captionof{figure}{\textbf{4-PLAYER SCENARIO}}
-\end{minipage}
-\vspace{1em}
-\linebreak
-\begin{minipage}{0.4\paperwidth}
-  \centering
-  \includegraphics[width=0.38\paperwidth]{\maps/titans-5.png}
-  \captionof{figure}{\textbf{5-PLAYER SCENARIO}}
-\end{minipage}
-\begin{minipage}{0.4\paperwidth}
-  \centering
-  \includegraphics[width=0.38\paperwidth]{\maps/titans-6.png}
-  \captionof{figure}{\textbf{6-PLAYER SCENARIO}}
-\end{minipage}
+
+  \matrix [matrix anchor=north west, column sep=1em]
+  {
+    \node {\includegraphics[width=\textwidth/2-1em, height=\textheight/3-4em, keepaspectratio]{\maps/titans-1.png}}; &
+    \node {\includegraphics[width=\textwidth/2-1em, height=\textheight/3-4em, keepaspectratio]{\maps/titans-2.png}}; \\
+    \node {\textbf{1-PLAYER SCENARIO}}; &
+    \node {\textbf{2-PLAYER SCENARIO}}; \\[1em]
+    \node {\includegraphics[width=\textwidth/2-1em, height=\textheight/3-4em, keepaspectratio]{\maps/titans-3.png}}; &
+    \node {\includegraphics[width=\textwidth/2-1em, height=\textheight/3-4em, keepaspectratio]{\maps/titans-4.png}}; \\
+    \node {\textbf{3-PLAYER SCENARIO}}; &
+    \node {\textbf{4-PLAYER SCENARIO}}; \\[1em]
+    \node {\includegraphics[width=\textwidth/2-1em, height=\textheight/3-4em, keepaspectratio]{\maps/titans-5.png}}; &
+    \node {\includegraphics[width=\textwidth/2-1em, height=\textheight/3-4em, keepaspectratio]{\maps/titans-6.png}}; \\
+    \node {\textbf{5-PLAYER SCENARIO}}; &
+    \node {\textbf{6-PLAYER SCENARIO}}; \\
+  };
+\end{tikzpicture}
```

Comment:
I like what you've done here, although I do believe you could take advantage of the unused margins. You can see in the render that the distance between 5- and 6-player setup and the page bottom increased. I think you should the opposite. Make 5- and 6- larger, allowing yourself to move the mcloser to the bottom. You can do the same trick with 1- and 2-, making them smaller and closer to the page top. It will make those images a little closer to their actual scale.